filter/patterns_storage.go

```diff
@@ -62,6 +66,11 @@ func (storage *PatternStorage) ProcessIncomingMetric(lineBytes []byte) *moira.Ma
 		return nil
 	}
 
+	if parsedMetric.IsTooOld(maxTTL, storage.clock.Now()) {
+		storage.logger.Infof("metric %v is too old", parsedMetric.Name)
```

Comment:
Nit, but I'd suggest us extracting specific field values into its own separate log entries for better debug experience:
```
storage.logger.Clone().Str(moira.LogFieldNameMetricName, parsedMetric.Name).Info("metric is too old")
```
Since it's minor, I'm leaving it up to you

Reply:
Done, furthermore added metric timestamp :)